eigentrust-cli/src/cli.rs

```diff
@@ -39,10 +43,14 @@ pub enum Mode {
 	Bandada(BandadaData),
 	/// Deploy the contracts.
 	Deploy,
+	/// Generate KZG parameters
+	KZGParams(KZGParamsData),
 	/// Calculate the global scores from the saved attestations.
 	LocalScores,
 	/// Generate the proofs.
 	Proof,
+	/// Generate proving key
+	ProvingKey,
```

Comment:
This should have some indication that it is for EigenTrust circuit
Different command needs to be made for Threshold circuit